src/content/docs/Products/SplashKit/Splashkit Expansion/Documentation/onboarding-guide.mdx

```diff
@@ -0,0 +1,205 @@
+---
+title: SplashKit Expansion Onboarding Guide
+sidebar:
+  label: SplashKit Expansion Onboarding Guide
+  order: 1
+---
+
+## Introduction
+
+This guide will cover all the steps required to get contributing to the splashkit-core repository.
+Feel free to skip steps which you have already completed or are familiar with.
+
+## Installing WSL
```

Comment:
Here it would be good to link to - https://splashkit.io/installation/windows-wsl/ which has the full WSL and splashkit guide.